docs/getting-started/gs-faqs.md

```diff
@@ -102,26 +102,31 @@ For details, see [Recommended Token Refresh Frequency](../ref-info/ref-tokens.md
 
 #### How can I test the refresh token workflow?
 
-You can use the `refresh-optout@example.com` email address to test your token refresh workflow. Using this email address in a request always generates an identity response with a `refresh_token` that results in a logout response.
+You can use the `refresh-optout@example.com` email address or the `+00000000002` phone number to test your token refresh workflow. Using either parameter value in a request always generates an identity response with a `refresh_token` that results in a logout response.
 
 The procedure is a little different depending on whether or not you are using an SDK.
 
 ##### With SDK:
 
-1. Send a [POST&nbsp;/token/generate](../endpoints/post-token-generate.md) request using one of the following values:
+1. Depending on whether the personal data is an email address or a phone number, send a [POST&nbsp;/token/generate](../endpoints/post-token-generate.md) request using one of the following values:
     - The `refresh-optout@example.com` as the `email` value.
     - The hash of `refresh-optout@example.com` as the `email_hash` value. 
-2. Wait until the SDK's [background auto-refresh](../sdks/sdk-ref-javascript.md#background-token-auto-refresh) attempts to refresh the advertising token (this can take several hours) and observe the refresh attempt fail with the OPTOUT status. At this point the SDK also clears the first-party cookie.
+    - The `+00000000002` as the `phone` value.
+    - The hash of `+00000000002` as the `phone_hash` value.
```

Comment:
do we know what this hash is?  is it somewhere else in the docs?  would maybe be nice to include that here rather than making them figure it out themselves.

Reply:
@ssundahlTTD as discussed: APIDOCS-2766.